src/Commands/StartCommand.php

```diff
@@ -23,6 +23,8 @@ class StartCommand extends Command implements SignalableCommandInterface
                     {--task-workers=auto : The number of task workers that should be available to handle tasks}
                     {--max-requests=500 : The number of requests to process before reloading the server}
                     {--rr-config= : The path to the RoadRunner .rr.yaml file}
+                    {--frankenphp-config= : The path to the FrankenPHP Caddyfile file}
+                    {--https : Enable HTTPS, HTTP/2 and HTTP/3, automatically generate and renew certificates}
```

Comment:
Maybe it is worth adding a note that the `--https` option only applies to FrakenPHP?